package.json

```diff
@@ -45,7 +45,7 @@
     "lint": "yarn lint:js && yarn lint:style",
     "lint:ci": "yarn lint:js --quiet && yarn lint:style",
     "lint:js": "eslint --cache  --report-unused-disable-directives \"**/*.{js,jsx,ts,tsx,mjs}\"",
-    "lint:style": "stylelint \"**/*.css\"",
+    "lint:style": "stylelint --fix \"**/*.css\"",
```

Comment:
Maybe it shouldn't include the `--fix` since it is used for checking, but the `--fix` is used in the Husky hook

Reply:
I'd still like to keep this script solely for checking because it's run in CI